src/opera_disp_tms/find_california_dataset.py

```diff
@@ -0,0 +1,268 @@
+        if frame in ASC_FRAMES:
+            orbit_pass = 'ASCENDING'
+        elif frame in DES_FRAMES:
+            orbit_pass = 'DESCENDING'
+        else:
+            orbit_pass = 'UNKNOWN'
+        url_base = 'https://cumulus-test.asf.alaska.edu/RTC/OPERA-S1/OPERA_L3_DISP-S1_PROVISIONAL_V0'
+        url = f'{url_base}/{scene_name}/{scene_name}.nc'
+        s3_base = 's3://asf-cumulus-test-opera-products/OPERA_L3_DISP-S1_PROVISIONAL_V0'
+        s3_uri = f'{s3_base}/{scene_name}/{scene_name}.nc'
+        reference_date = datetime.strptime(scene_name.split('_')[-4], DATE_FORMAT)
+        secondary_date = datetime.strptime(scene_name.split('_')[-3], DATE_FORMAT)
+        creation_date = datetime.strptime(scene_name.split('_')[-1], DATE_FORMAT)
+        return cls(
+            scene_name=scene_name,
+            frame=frame,
+            orbit_pass=orbit_pass,
+            url=url,
+            s3_uri=s3_uri,
+            reference_date=reference_date,
+            secondary_date=secondary_date,
+            creation_date=creation_date,
+        )
+
+    @classmethod
+    def from_tuple(cls, tup: Tuple) -> 'Granule':
+        """Create a Granule object from a tuple.
+
+        Args:
+            tup: The tuple to create the Granule from.
+
+        Returns:
+            A Granule object created from the tuple.
+        """
+        name, frame, orbit_pass, url, s3_uri, reference_date, secondary_date, creation_date = tup
+        return cls(
+            scene_name=name,
+            frame=int(frame),
+            orbit_pass=orbit_pass,
+            url=url,
+            s3_uri=s3_uri,
+            reference_date=datetime.strptime(reference_date, DATE_FORMAT),
+            secondary_date=datetime.strptime(secondary_date, DATE_FORMAT),
+            creation_date=datetime.strptime(creation_date, DATE_FORMAT),
+        )
+
+    def to_tuple(self):
+        return (
+            self.scene_name,
+            self.frame,
+            self.orbit_pass,
+            self.url,
+            self.s3_uri,
+            datetime.strftime(self.reference_date, DATE_FORMAT),
+            datetime.strftime(self.secondary_date, DATE_FORMAT),
+            datetime.strftime(self.creation_date, DATE_FORMAT),
+        )
+
+
+def find_test_data(desired_version: str = '0.4') -> Granule:
+    """Find all OPERA L3 DISP S1 PROVISIONAL granules created for a specific version.
+
+    Args:
+        desired_version: The version of the granules to search for.
+
+    Returns:
+        A list of granules that match the search criteria.
+    """
+    session = asf.ASFSession(edl_host='uat.urs.earthdata.nasa.gov')
+    disp_opts = asf.ASFSearchOptions(
+        session=session, host='cmr.uat.earthdata.nasa.gov', shortName='OPERA_L3_DISP-S1_PROVISIONAL_V0'
+    )
+    disp_opts.maxResults = None
+    cmr_keywords = [('options[readable_granule_name][pattern]', 'true')]
+    granule_name_wildcard = [f'*_v{desired_version}_*']
+    granule_name_wildcard = ['*']
```

Comment:
`granule_name_wildcard` is overwritten in line 190. Is that the right wildcard? If so, delete line 189

Reply:
Done!